google/cloud/bigtable/backup.py

```diff
@@ -0,0 +1,342 @@
+        )
+
+    @property
+    def cluster(self):
+        return self._cluster
+
+    @cluster.setter
+    def cluster(self, cluster_id):
+        self._cluster = cluster_id
+
+    @property
+    def parent(self):
+        if not self._parent and self._cluster:
+            self._parent = self._instance.name + "/clusters/" + self._cluster
+        return self._parent
+
+    @property
+    def source_table(self):
+        """The full name of the Table from which this Backup is created.
+        The table name is of the form
+
+            ``"projects/../instances/../tables/{source_table}"``
+
+        :rtype: str
+        :returns: The Table name.
+        """
+        if not self._source_table and self.table_id:
+            self._source_table = "{}/tables/{}".format(
+                self._instance.name, self.table_id
+            )
+        return self._source_table
+
+    @property
+    def expire_time(self):
+        """Expiration time used in the creation requests.
+
+        :rtype: :class:`datetime.datetime`
+        :returns: A 'datetime' object representing the expiration time of
+                  this Backup.
+        """
+        return self._expire_time
+
+    @expire_time.setter
+    def expire_time(self, new_expire_time):
+        self._expire_time = new_expire_time
+
+    @property
+    def start_time(self):
+        """The time this Backup was started.
+
+        :rtype: :class:`datetime.datetime`
+        :returns: A 'datetime' object representing the time when the creation
+                  of this Backup had started.
+        """
+        return self._start_time
+
+    @property
+    def end_time(self):
+        """The time this Backup was finished.
+
+        :rtype: :class:`datetime.datetime`
+        :returns: A 'datetime' object representing the time when the creation
+                  of this Backup was finished.
+        """
+        return self._end_time
+
+    @property
+    def size_bytes(self):
+        """The size of this Backup, in bytes.
+
+        :rtype: int
+        :returns: The size of this Backup, in bytes.
+        """
+        return self._size_bytes
+
+    @property
+    def state(self):
+        """ The current state of this Backup.
+
+        :rtype: :class:`~google.cloud.bigtable_admin_v2.gapic.enums.Backup.State`
+        :returns: The current state of this Backup.
+        """
+        return self._state
+
+    @classmethod
+    def from_pb(cls, backup_pb, instance):
+        """Creates a Backup instance from a protobuf message.
+
+        :type backup_pb: :class:`table_pb2.Backup`
+        :param backup_pb: A Backup protobuf object.
+
+        :type instance: :class:`Instance <google.cloud.bigtable.instance.Instance>`
+        :param instance: The Instance that owns the Backup.
+
+        :rtype: :class:`~google.cloud.bigtable.backup.Backup`
+        :returns: The backup parsed from the protobuf response.
+        :raises ValueError:
```

Comment:
`:raises: ValueError` - colon should be set right after the word "raises" here and in the methods lower

Reply:
I wonder 
:+1: I wonder why Blacken is not configured to catch such errors.